manifests/globals.pp

```diff
@@ -15,13 +15,20 @@
 #   The mode specifies the specifics in paths for the various RedHat SCL environments so that the module is configured
 #   correctly on their pathnames.
 #
+# [*flavor*]
+#   Flavor of PHP, either 'community' or 'zend'.
+#
+# [*zend_creds*]
+#   Hash of ZendPHP repo credentials; {username => '<USERNAME>', password => '<PASSWORD>'}
+#
 
 class php::globals (
-
   Optional[Pattern[/^(rh-)?(php)?[578](\.)?[0-9]/]] $php_version = undef,
-  Optional[Stdlib::Absolutepath] $config_root   = undef,
-  Optional[Stdlib::Absolutepath] $fpm_pid_file  = undef,
-  Optional[Enum['rhscl', 'remi']] $rhscl_mode   = undef,
+  Optional[Stdlib::Absolutepath] $config_root                    = undef,
+  Optional[Stdlib::Absolutepath] $fpm_pid_file                   = undef,
+  Optional[Enum['rhscl', 'remi']] $rhscl_mode                    = undef,
+  Optional[Enum['community', 'zend']] $flavor                    = undef,
```

Comment:
```suggestion
  Enum['community', 'zend'] $flavor = 'community',
```

IMO if the default in the `case` statements below is `community`, this parameter should to the same.